haddock.cabal

```diff
@@ -12,9 +12,10 @@ bug-reports:          https://github.com/haskell/haddock/issues
 copyright:            (c) Simon Marlow, David Waern
 category:             Documentation
 build-type:           Simple
-cabal-version:        >= 1.10
+cabal-version:        >= 2.0
```

Comment:
I've tweaked `cabal-version` in GHC HEAD, and ran a `./validate` and it works just fine. So false alarm.